GherkinSyncTool/Program.cs

```diff
@@ -74,11 +72,33 @@ private static int Main(string[] args)
             return 0;
         }
 
-        private static void RegisterSynchronizer(object obj)
+        private static void ParseCommandLineArgs(string[] args)
+        {
+            var parser = new Parser(settings => settings.HelpWriter = null);
+            var parserResult = parser.ParseArguments(args, CommandLineParserHelper.LoadCliVerbsTypes());
+            parserResult
+                .WithParsed(RegisterSynchronizer)
+                .WithNotParsed(errs => CommandLineParserHelper.DisplayHelp(parserResult, errs));
+        }
+
+        private static void RegisterSynchronizer(object options)
         {
+            var syncOptions = (SyncOptions) options;
+
+            if (syncOptions.Testrail)
```

Comment:
I guess we better use switch statement here if we imply several test environments

Reply:
Done